### PR TITLE
Additional Tab For Release

### PR DESCRIPTION
**Pull request by Mussuranae**
Additional tab for dashboard view

### CODE REVIEW

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html [hunk 1]
```diff
@@ -8,19 +8,26 @@ <h6>
...
-          <missing-control [control]="release.get('media')" [link]="link" scope="MEDIAS"></missing-control> -
-          <missing-control [control]="release.get('date')" [link]="link">
-            <ng-template let-date>
-              {{ date | date: 'dd/MM/yyyy' }}
-            </ng-template>
-          </missing-control>
-        </ng-container> -
-      </ng-template>
-    </missing-control>
+    <ul>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:11]: only one item (li) in your list (ul) is this normal?

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html [hunk 2]
```diff
@@ -31,33 +38,39 @@ <h6>
...
-            </ng-template>
-          </missing-control>
-          &nbsp;
-          <missing-control [control]="office.get('value')">
-            <ng-template let-value>{{ value | number }}</ng-template>
-          </missing-control>
-        </ng-container>
-      </ng-template>
-    </missing-control>
+    <ul>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:41]: only one item (li) in your list (ul) is this normal?

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html [hunk 3]
```diff
@@ -88,7 +123,11 @@ <h6>
       <mat-icon svgIcon="pencil" color="primary"></mat-icon>
     </a>
   </h6>
-  <missing-control [control]="movie.certifications" [link]="link" fragments="qualifications">
-    <ng-template let-certs>{{ certs | translateSlug: 'CERTIFICATIONS' }}</ng-template>
-  </missing-control>
-</article>
\ No newline at end of file
+  <ul>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:126]: same

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html [hunk 4]
```diff
@@ -66,20 +79,42 @@ <h6>
...
+            </li>
+          </ng-container>
+        </ng-template>
+      </missing-control>
+    </ul>
 
   </article>
 </ng-container>
 
+<!-- BUDGET RANGE -->
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:101]: same

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html [hunk 5]
```diff
@@ -66,20 +79,42 @@ <h6>
...
-      <ng-template let-ratings="control">
-        <ng-container *ngFor="let rating of ratings.controls">
-          <missing-control [control]="rating.get('value')" [link]="link" fragment="rating"></missing-control>
-          <span>&nbsp;</span>
-          <missing-control [control]="rating.get('country')" [link]="link" fragment="rating" scope="TERRITORIES">
-          </missing-control>
-        </ng-container>
-      </ng-template>
-    </missing-control>
+    <ul>
```
Comment (phpgeekfr) [inline libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:82]: only one item (li) in your list (ul) is this normal?

**Review by phpgeekfr — APPROVED** [review]

File: libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html (hunk 2 above)
Comment (Mussuranae) [reply libs/movie/src/lib/movie/form/summary/additional-information/additional-information.component.html:41]: the `li` tag is inside the ngFor ;)

**Pull request merged**